man/ocitools-validate.1.md

```diff
@@ -18,6 +21,9 @@ Validate an OCI bundle
 **--path=PATH
    Path to bundle
 
+**--hooks-check**
```

Comment:
@wking  I can't agree with you.
We are not sure if there will be other options to validate  going to enable/disable particular checks, as hooks is a lit sepcial.
By now, there is only hooks need to enablle/disable a check and I think an obvious option name is better.